src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html"><img src="./images/logo.png" alt="Moyo"></a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item"><a href="#apple" class="nav_link is-active">Apple</a></li>
+          <li class="nav_item"><a href="#samsung" class="nav_link">Samsung</a></li>
+          <li class="nav_item"><a href="#smartphones" class="nav_link">Smartphones</a></li>
+          <li class="nav_item">
+            <a href="#laptops&computers" class="nav_link"
+            data-qa="hover" >Laptops & Computers</a>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
13. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.